### DISCUSSION TITLE
No update cli option (kali)

### DISCUSSION THREAD

**Discussion by ehsandeep** [projectdiscovery/nuclei#2151] [General]
Nuclei installed on **[kali](https://www.kali.org/tools/nuclei/)** using `apt install nuclei` do not have update option as the option is patched [here](https://gitlab.com/kalilinux/packages/nuclei/-/blob/kali/master/debian/patches/Disable-update.patch).

As a result:
- Nuclei engine can not be updated using `nuclei -update` option
- Nuclei engine can not be updated using `apt update nuclei` as the releases are [not maintained](https://gitlab.com/kalilinux/packages/nuclei/).

If you are using nuclei on kali, and looking to update to the latest version, try to install nuclei using other available installation options as detailed here - https://nuclei.projectdiscovery.io/nuclei/get-started/#nuclei-installation after removing the currently installed version.

**Comment by ehsandeep** [comment]
Follow up issue for release maintenance / process: https://gitlab.com/kalilinux/packages/nuclei/-/issues/1

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
